content/exchange/artifacts/Windows.Events.HighPrivilegedLogon.yaml

```diff
@@ -0,0 +1,42 @@
+name: Windows.Events.HighPrivilegedLogon
```

Comment:
This one is probably better implemented as a sigma rule 

https://sigma.velocidex.com/docs/artifacts/windows.hayabusa.monitoring/